spec/factory_specs/subject_spec.rb

```diff
@@ -1,12 +1,24 @@
 require 'rails_helper'
+describe "Subject Factory" do
+  describe "Subject 'Further Education' Factory" do
+    let(:subject) { create(:futher_education_subject) }
 
-describe "Subject 'Further Education' Factory" do
-  let(:subject) { create(:futher_education_subject) }
+    it "created subject" do
+      expect(subject).to be_instance_of(Subject)
+      expect(subject).to be_valid
+      expect(subject.subject_name).to eq 'Further Education'
+      expect(subject.in?(Subject.further_education)).to be true
+    end
+  end
+
+  describe "Subject 'Send' Factory" do
+    let(:subject) { create(:send_subject) }
 
-  it "created subject" do
-    expect(subject).to be_instance_of(Subject)
-    expect(subject).to be_valid
-    expect(subject.subject_name).to eq 'Further Education'
-    expect(subject.in?(Subject.further_education)).to be true
+    it "created subject" do
+      expect(subject).to be_instance_of(Subject)
+      expect(subject).to be_valid
+      expect(subject.subject_name).to eq 'Special Educational Needs'
+      expect(subject.subject_code).to eq 'U3'
+    end
```

Comment:
Same comment re: one-liners.

Reply:
does that mean you going to let me keep factory